app/assets/images/payment_icons/afterpay_paynl_version.svg

```diff
@@ -0,0 +1 @@
32l-.03.15v.05c-.11.47-.49.72-.97.65-.31-.05-.73-.21-.73-1.07v-.06h.94c.11 0 .46 0 .46-.34 0-.17-.13-.31-.28-.32-.07-.01-.13-.01-.2-.01h-.92v-1.31c0-.12-.01-.21-.05-.28a.36.36 0 00-.21-.17c-.09-.02-.19 0-.27.05-.14.09-.16.25-.16.41v1.95c0 1.11.4 1.67 1.3 1.81.1.02.21.03.31.03.71 0 1.3-.44 1.49-1.14.01-.06.02-.11.03-.16.04-.2.03-.31-.03-.42zm4.61-.93c-.3-1.13-1.45-1.78-2.62-1.49-.6.15-1.09.51-1.37 1.02s-.34 1.12-.15 1.71c.28.87 1.09 1.42 2.01 1.42.17 0 .34-.02.51-.06.26-.06.51-.17.74-.31.17-.11.31-.24.39-.33.12-.14.19-.25.17-.39a.36.36 0 00-.13-.23c-.08-.06-.17-.08-.26-.06h-.01a.66.66 0 00-.22.12l-.06.08-.09.08c-.2.2-.44.34-.7.39-.7.16-1.4-.21-1.65-.88-.02-.06-.03-.11-.05-.16h3.09l.07-.01h.1c.11-.01.2-.05.25-.13.11-.13.1-.35-.02-.77zm-3.45.12c.24-.76 1.01-1.18 1.78-.95.49.14.86.54.99 1.03.01.04.02.08.02.11l-2.84-.02c.01-.06.03-.11.05-.17zm7.09-.43a.75.75 0 00-.05-.16l-.01-.04c-.25-.68-.81-1.08-1.51-1.08-.12 0-.23.01-.34.03-.56.12-1.22.52-1.22 1.74v1.92c0 .14.03.22.07.28.04.08.12.14.21.17.02 0 .05.01.08.01.07 0 .14-.02.19-.06.15-.1.15-.27.15-.41v-1.9c0-.64.2-.95.67-1.05.32-.07.63.03.82.26.09.11.15.22.17.32.07.34.11.44.16.51l.01.02c.1.12.29.15.44.08.23-.13.21-.41.16-.64zm4.11-1.84c-.35-.42-.87-.66-1.44-.65-1.06.01-1.78.78-1.78 1.92v2.99c0 .14.03.22.07.28.04.08.12.14.21.17.02 0 .05.01.08.01.07 0 .14-.02.19-.06.15-.1.17-.26.17-.41v-1.35h1.03c.98 0 1.64-.51 1.83-1.4.11-.55-.02-1.09-.36-1.5zm-.33 1.13c-.01.64-.48 1.08-1.14 1.08h-.01l-1.01-.01v-.99c0-.44.07-.65.28-.87.21-.22.48-.32.81-.32.63.01 1.07.47 1.07 1.11zm5.17 1.36c-.01-.17-.03-.32-.08-.46-.15-.51-.49-.93-.97-1.2a2.18 2.18 0 00-1.68-.17c-.75.22-1.32.83-1.47 1.59-.23 1.14.41 2.21 1.49 2.49.2.05.4.08.6.08.3 0 .59-.06.86-.18.22-.1.41-.23.56-.39v.01c0 .13 0 .2.04.28.04.08.12.14.21.17.09.02.19 0 .27-.05.15-.1.17-.26.17-.41v-1.76zm-1.77 1.61c-.78.18-1.5-.25-1.71-1.01-.03-.1-.05-.2-.06-.3-.04-.67.36-1.28.99-1.48.15-.05.31-.07.46-.07.29 0 .57.08.81.25.36.25.58.67.59 1.14.03.72-.4 1.31-1.08 1.47zm5.52-3.24l-.15.07.15-.07a.337.337 0 00-.22-.17c-.09-.02-.19 0-.27.05-.14.09-.16.25-.16.41v1.9c0 .37-.11.67-.32.85-.18.16-.42.22-.7.18-.48-.07-.75-.41-.75-.97v-1.97c0-.12-.01-.2-.05-.28a.36.36 0 00-.21-.17c-.09-.02-.19 0-.27.05-.15.1-.17.26-.17.41v1.97c0 .79.36 1.35 1.01 1.57.51.17 1.06.09 1.44-.19v.01c-.06.42-.27.67-.65.76a.79.79 0 01-.57-.04c-.13-.07-.23-.14-.28-.22l-.1-.13-.07-.08c-.04-.06-.08-.08-.09-.1l-.02-.02-.02-.01a.311.311 0 00-.31-.01.36.36 0 00-.2.25c-.02.15.04.28.15.44a1.517 1.517 0 001.25.63c.15 0 .3-.02.45-.06.76-.19 1.18-.8 1.18-1.71v-3.09c.01-.13-.03-.21-.05-.26zm-24.47.98a.75.75 0 00-.05-.16l-.01-.04c-.25-.68-.81-1.08-1.51-1.08-.13 0-.23.01-.34.03-.56.12-1.22.52-1.22 1.74v1.92c0 .15.04.23.07.28.04.08.12.14.21.17.02 0 .05.01.08.01.07 0 .14-.02.19-.06.14-.09.16-.25.16-.41v-.45l-.01-.8h1.01c.33 0 .41-.17.42-.32 0-.12-.04-.2-.08-.24-.07-.08-.18-.11-.34-.11H8.04c0-.62.21-.93.67-1.03.32-.07.63.03.82.26.1.12.16.22.18.32.07.34.11.44.16.51l.01.02c.1.12.29.15.44.08.22-.13.2-.41.15-.64z" class="st2"/></svg>
```

Comment:
Can we remove `class="st2"`? It is not used.

Reply:
It is removed now